src/Math-Matrix/PMMatrix.class.st

```diff
@@ -824,6 +824,20 @@ PMMatrix >> qrFactorizationWithPivoting [
 	^ Array with: q with: r with: pivot
 ]
 
+{ #category : #operation }
+PMMatrix >> raisedTo: anInteger [
+
+	" Answers the receiver raised to a power anInteger.
+	 If anInteger is negative, inverse of the receiver is raised to the absolute value of anInteger."
+	self assert: self numberOfRows = self numberOfColumns description: 'Matrix should be square'.
+	
+	 anInteger <= 0 ifTrue: [ 
+		anInteger = 0 ifTrue: [^ PMMatrix identity: self numberOfRows ]
```

Comment:
Nested ifs can impact comprehensibility. Perhaps, makes this a guard clause all on its own for a power = 0. Assuming you've not considered this already, is there a way we may avoid chains ifTrue/ifFalse?

Reply:
@hemalvarambhia I have reformatted it now. Can you please check it?